packages/spacecat-shared-gpt-client/src/clients/firefall-client.js

```diff
@@ -89,15 +96,10 @@ export default class FirefallClient {
     this.log.debug(`${message}: took ${duration}ms`);
   }
 
-  async #submitJob(prompt) {
+  async #submitPrompt(prompt, body, path) {
     const apiAuth = await this.#getApiAuth();
 
-    const body = JSON.stringify({
-      input: prompt,
-      capability_name: this.config.capabilityName,
-    });
-
-    const url = createUrl(`${this.config.apiEndpoint}/v2/capability_execution/job`);
+    const url = createUrl(`${this.config.apiEndpoint}${path}`);
```

Comment:
Should the firefall-client export an array of valid Firefall api endpoints that can be used with the config? Otherwise every user of the client will need to maintain their own set of endpoints.

Reply:
IIUC, every user of the firefall client must provide their own Firefall endpoint URL.

eg. `FIREFALL_API_ENDPOINT: 'https://api.firefall.example.com',`

AFAIK, there are only 2 possible URLs so why can't the firefall client manage them?

```
Stage: [firefall-stage.adobe.io](http://firefall-stage.adobe.io/)
Production: [firefall.adobe.io](http://firefall.adobe.io/)
```

Reply:
If the endpoint changes any users of the client would have to be be updated anyway. Any string could still be used as the FIREFALL_API_ENDPOINT but exporting the default URL values would just be helpful.

I was thinking of exporting a frozen firefall config object.

```
export const firefallEndpointConfig = Object.freeze({
  stage: 'http://firefall-stage.adobe.io',
  prod: 'http://firefall.adobe.io',
});
```

Where? I guess it could be a static object of the `FirefallClient` class.

Reply:
the URL should be controlled via env var from the AWS secrets relevant to the dev/stage/prod environment.

Reply:
I'm reading that as letting the AWS secrets control the endpoint, and not having any exported config's specifying the possible endpoints.
So I'll remove them.  Let me know if I misunderstood.